src/tasks/VersionFeedTask.ts

```diff
@@ -220,7 +222,7 @@ export default class VersionFeedTask extends Task {
 		};
 
 		try {
-			versionIssueCounts = await MojiraBot.jira.projectVersions.getVersionsRelatedIssuesCount( {
+			versionIssueCounts = await MojiraBot.jira.projectVersions.getVersionRelatedIssues( {
```

Comment:
The documentation for jira.js seems to imply that this method also returns the usage of this version in custom fields. Would this allow it to continue to comply with the type defined above?
<img width="785" alt="Screen Shot 2022-01-07 at 11 35 24 PM" src="https://user-images.githubusercontent.com/68699877/148631689-286dce9e-4a6f-445e-8b04-29a7cf7aaf2c.png">